### PR TITLE
Feature javascript functionality

### PR DESCRIPTION
**Pull request by TimNguyen21**
Javascript Update

- Modification and update to previous codes
- Added new functions

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -17,16 +17,39 @@ var maxRange = document.querySelector('#max-range');
...
+}
+
+function resetForm() {
+  minSpan.innerText = "?";
+  maxSpan.innerText = "?";
+  oneName.value = "";
+  twoName.value = "";
+  minRange.value = "";
+  maxRange.value = "";
+
```
Comment (jonathan-tschida) [inline main.js:51]: keep an eye on unnecessary whitespace like this

File: main.js [hunk 2]
```diff
@@ -81,23 +104,49 @@ function updateLatestGuess() {
...
+  minRange.value = "";
+  maxRange.value = "";
+  updateButton.disabled = true;
+}
+
 function checkGuess(guessInput, feedbackMessage) {
   var currentGuess = parseInt(guessInput.value);
+  var tooHigh = "that's too high";
+  var tooLow = "that's too low";
+  var perfectGuess = "BOOM!";
```
Comment (jonathan-tschida) [inline main.js:122]: Is there a reason you made these variables?  This just seems cluttered to me

File: main.js [hunk 3]
```diff
@@ -17,16 +17,39 @@ var maxRange = document.querySelector('#max-range');
...
 clearButton.disabled = true;
 resetButton.disabled = true;
+updateButton.disabled = true;
 
 document.addEventListener('keyup', enableSubmit);
 document.addEventListener('keyup', enableClear);
+document.addEventListener('keyup', checkRange);
 clearButton.addEventListener('click', clearInputs);
 submitButton.addEventListener('click', updateLatestGuess);
 updateButton.addEventListener('click', updateRange);
```
Comment (jonathan-tschida) [inline main.js:36]: This doesn't do anything now, if you remove the updateRange function

File: main.js (hunk 2 above)
Comment (TimNguyen21) [reply main.js:122]: I was thinking if a person guess the right answer, the perfectGuess variable will be used to add the Winner's Output card on the right side on another function.  If there is another of doing this, then you can change it back to the original. I don't mind.  I wasn't quite sure of how to start with that part, so I did that is my first idea.

File: main.js (hunk 1 above)
Comment (TimNguyen21) [reply main.js:51]: Thanks.  I didn't remove any whitespace because I know that we will clean up in the future.  I'll try to keep a habit of cleaning up whitespaced will I code.

File: main.js (hunk 2 above)
Comment (jonathan-tschida) [reply main.js:122]: These variables are stored locally within the function, so they can only be used by the function.  Let's just worry about one feature at a time.

File: main.js (hunk 3 above)
Comment (TimNguyen21) [reply main.js:36]: There are some functions or variables throughout that I made, but eventually not using.  So, that is why there might be some parts that are unresponsive.  I was going to clean it up when we do our review and clean up.

File: main.js (hunk 2 above)
Comment (TimNguyen21) [reply main.js:122]: Thanks for letting me know that.  That will be helpful for me in the future.  I'm still working on my javascript skills. =)

File: main.js (hunk 3 above)
Comment (jonathan-tschida) [reply main.js:36]: Ignore this anyway, I just realized you moved the updateRange function further up the code. My mistake!

Comment (TimNguyen21) [reply main.js:36]: No problem!

File: main.js [hunk 4]
```diff
@@ -81,23 +104,49 @@ function updateLatestGuess() {
...
+    errorIcon.style.visibility = 'hidden';
+    maxRange.style.border = '1px solid #d0d2d3';
+  }
+  if (min < max) {
+    updateButton.disabled = false;
+    updateButton.style.backgroundColor = '#6e6e6e';
+  } else {
+    updateButton.disabled = true;
+    updateButton.style.backgroundColor = '#d0d2d3';
+  }
```
Comment (jonathan-tschida) [inline main.js:151]: You can  combine the two if else statements into one, since they're basically saying the same thing but in reverse.  If the maximum is smaller than or equal to the min then the button will be disabled, and if it is greater than the min the button will be enabled.  Also you don't need to style the button in JS, the :disabled class in CSS will do that for us.

`if (min >= max) {`
    `errorIcon.style.visibility = 'visible';`
    `maxRange.style.borderColor = '#dd1972';`
    `updateButton.disabled = true;`
    `updateButton.style.backgroundColor = '#d0d2d3';`
 ` } else {`
    `errorIcon.style.visibility = 'hidden';`
    `maxRange.style.border = '1px solid #d0d2d3';`
    `updateButton.disabled = false;`
    `updateButton.style.backgroundColor = '#6e6e6e';`
`  }`

**Review by jonathan-tschida — APPROVED** [review]
Looks good for the most part, just needs a few changes when we come back and clean up the code
